js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.js

```diff
@@ -55,170 +51,151 @@ const PreLaunchChecklist = ( props ) => {
-												'google-listings-and-ads'
-											) }
-										</span>
-										<HelpPopover id="contact_info_visible">
-											{ createInterpolateElement(
-												__(
-													'Your website must display sufficient and accurate contact information to your customers, including a telephone number and/or email. <link>Read more</link>',
-													'google-listings-and-ads'
-												),
-												{
-													link: (
-														<AppDocumentationLink
-															context="setup-mc-checklist"
-															linkId="check-phone-numbers"
-															href="https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#contact-info"
-														/>
-													),
-												}
-											) }
-										</HelpPopover>
-									</span>
-								}
-								{ ...getInputProps( 'contact_info_visible' ) }
-							/>
+							<PreLaunchCheckItem
+								formProps={ formProps }
+								fieldName="website_live"
+								firstPersonTitle={ __(
+									'My store is live and accessible to all users',
+									'google-listings-and-ads'
+								) }
+								secondPersonTitle={ __(
+									'Confirm your store is live and accessible to all users',
+									'google-listings-and-ads'
+								) }
+							>
+								{ __(
+									'Your Merchant Center account can be suspended if your store is not functional. Ensure that your domain and product pages are not leading to an under construction webpage, or an error page that displays a status code beginning with a 4 or 5 (such as a 405 error).',
+									'google-listings-and-ads'
+								) }
+								<AppDocumentationLink
+									context="setup-mc-checklist"
+									linkId="check-website-is-live"
+									type="external"
+									href="https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#store-is-live"
+								>
+									{ __(
+										'Learn more about common landing page issues and how to fix them',
+										'google-listings-and-ads'
+									) }
+								</AppDocumentationLink>
+							</PreLaunchCheckItem>
+							<PreLaunchCheckItem
+								formProps={ formProps }
+								fieldName="payment_methods_visible"
+								firstPersonTitle={ __(
+									'I have a complete checkout process',
+									'google-listings-and-ads'
+								) }
+								secondPersonTitle={ __(
+									'Confirm you have a complete checkout process',
+									'google-listings-and-ads'
+								) }
+							>
+								{ __(
+									'Ensure that all customers are able to complete the full checkout process on your site with an eligible payment method. Include a confirmation of the purchase after completion of the checkout process. ',
+									'google-listings-and-ads'
+								) }
+								<AppDocumentationLink
+									context="setup-mc-checklist"
+									linkId="check-payment-methods-visible"
+									type="external"
+									href="https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#payment-methods"
```

Comment:
This link was changed from https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#complete-checkout to https://woocommerce.com/document/google-listings-and-ads/compliance-policy/#payment-methods in 18e9ff25e8c91f23f096d163ebe17e05e8275447. However, it seems not to be an expected change.

Reply:
Thanks for the offline suggestions. Updated.